### DISCUSSION TITLE
Edge - Blank Windows

### DISCUSSION THREAD

**Discussion by ChrisMch** [cypress-io/cypress#26163] [General]
Hi,
I saw this issue many time, but never solved.
I will try to describe my problem as much as I can. Maybe we could find a solution together.
So...

**Versions**
Cypress Version : 11.0.0
Browser : Edge Beta v111
Behind a proxy

**Problem**
When I run `npx cypress open` and choose Edge Beta this message appear in the console:
`Still waiting to connect to Edge beta, retrying in 1 second (attempt 18/62)`
And in the browser I have a blank window...

**Solution tested** 
Try to add this ID "caljajdfkjjjdehjdoimjkkakekklcck" in key registry `HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Edge\ExtensionInstallAllowlist` (Extension ID found in Cypress Documentation https://docs.cypress.io/guides/references/troubleshooting#Allow-the-Cypress-Chrome-extension). It's chrome extension but i don't find the ID for Edge. And Edge is based on Chromium so... it could be ok...

Everything is ok on Electron. 
Do you have any suggestion for this problem? Maybe Proxy configuration? or something to do with the ID extension? or other things?

Thanks a lot!

**Comment by ChrisMch** [comment]
Hi,

To complete my message, it's the same thing with Edge v110
![image](https://user-images.githubusercontent.com/40334121/226845692-7ac5e956-bf54-4a69-807d-c5ea0c8d4a81.png)

**Comment by ardakod** [comment]
@ChrisMch Hello I have the exact same problem I am behind a corporate proxy and cypress work on electron but not on edge. Have you found the cause of that please? I suspect that it comes from the proxy...

**Reply by ChrisMch** [reply]
Hi !
The problem was about the GPO but I can't tell you more..
I contacted the administrator of my company and they modified the GPO...
Hope you could find the solution for your case.
